### ISSUE TITLE
input not selected when focused using tab

### ISSUE THREAD

**Issue by gest01** [aguirrel/ng-currency#118]
### Description

When input has focus using tab key, the content of the input is not selected. Instead, the cursor is set to the end of the input value.

### Expected outcome

As soon the input is focused (using the tab key), the content of the input should be fully selected

### Actual outcome

 the cursor is set to the end of the input value.
### Steps to reproduce

1. https://jsbin.com/niwugeyice/1/edit?html,output
2. Click with the mouse into input 'txt1'
3. press tab
4. txt2 receives the focus, but nothing is selected

### Browsers Affected
- [x ] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ x] IE 11
- [ ] IE 10

Works in Firefox and Edge

**Comment by aguirrel** [comment]
Yes you are right.  It's a side effect for customize 'focus' evento to convert from currency view to only numeric view.  I will check it.  Thanks a lot!

**Comment by ztgluis** [comment]
Might not be the most elegant solution, but for now I found a workaround by adding a decorator to the link function on my app config:

```
.config(function($provide) {
        $provide.decorator('ngCurrencyDirective', function($delegate) {
            var directive = $delegate[0];
            directive.compile = function(){
                return function(scope, element, attrs){
                    directive.link.apply(this, arguments);
                    element.on('focus',function(){
                        element.select();
                    });
                };
            };
            return $delegate;
        });
    })
```
